Cargo.toml

```diff
@@ -3,6 +3,7 @@
 members = [
     "pax-lang",
     "pax-core",
+    "pax-examples/*",
```

Comment:
By including `pax-examples` in the workspace, it is now possible to directly call `cargo run --bin pax-cli --path pax-examples/SOME_EXAMPLE` instead of relying on the `pax` script